dayzconfig.json

```diff
@@ -0,0 +1,474 @@
+[
+  {
+    "DisplayName": "Steam Query Port",
+    "Category": "Hidden Settings",
+    "Description": "",
+    "Keywords": "",
+    "FieldName": "$QueryPort",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "steamQueryPort",
+    "IncludeInCommandLine": false,
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "RCon Password",
+    "Category": "Hidden Settings",
+    "Description": "",
+    "Keywords": "",
+    "FieldName": "$RemoteAdminPassword",
+    "InputType": "password",
+    "IsFlagArgument": false,
+    "ParamFieldName": "passwordRcon",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {},
+    "Hidden": true
+  },
+  {
+    "DisplayName": "File Patching",
+    "Category": "DayZ - Miscellaneous",
+    "Description": "Ensures that only PBOs are loaded and NO unpacked data.",
+    "Keywords": "patch,patching,files,misc,pbo,data",
+    "FieldName": "filePatching",
+    "InputType": "checkbox",
+    "IsFlagArgument": true,
+    "ParamFieldName": "filePatching",
+    "IncludeInCommandLine": true,
+    "DefaultValue": "-filePatching",
+    "EnumValues": {
```

Comment:
Could you elaborate?

Reply:
I removed the enumvalues since they aren't needed, but not sure what you're pointing out about this either. It's a flag argument that forces only "official" packs are loaded it seems.